openslides/utils/rest_api.py

```diff
@@ -89,6 +90,86 @@ class ViewSet(PermissionMixin, _ViewSet):
     pass
 
 
+class RESTElement(metaclass=RegisterSubclasses):
+    """
+    Element of our REST api.
+
+    Each element bundels a viewset, a route and optionally a model together.
+    """
+
+    viewset = None
```

Comment:
Your implementation looks like that viewset never should be None. See line 168. So you have to ensure, that all subclasses define this attribute properly.